src/app/pages/common/ix-forms/components/ix-combobox2/ix-combobox2-provider.interface.ts

```diff
@@ -0,0 +1,9 @@
+import { Observable } from 'rxjs';
+import { Option } from '../../../../../interfaces/option.interface';
+
+export interface IxCombobox2Provider {
+  pageOffset: number;
```

Comment:
I am not sure that we need `pageOffset` in this interface. Seems like this should be something for provider to figure out internally.